### PR TITLE
Reworks the Build Your Own Shuttle Kit to be more appealing.

### PR DESCRIPTION
**Pull request by WJohn**
## About The Pull Request
This makes it so the BYOS starts with some construction materials, is a bit smaller to be less of a hideously huge project to work on, and is pressurized so more people can help with it.

Price has been increase to -2500 from -7500.

## Why It's Good For The Game
I often see the BYOS barely built, and even then usually only by one person who actually has a space suit. This often leaves it completely unprepared by the time the shuttle is actually called. This PR attempts to rectify that.

New shuttle:

## Changelog
:cl: WJohnston
tweak: Build Your Own Shuttle kit has been made smaller, comes with construction materials, and is pressurized.
balance: Lowered value from -7500 to -2500. It still doesn't come with a brig or console, so you cannot greentext antags or launch early.
/:cl:

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
i think people like to change the external walls too how about some crappy holowalls with 5 hp and no density

**Comment by WJohn** [comment]
I thought about leaving the shuttle depressurized so people could reconstruct the outside walls, but as it is now if people want to do that, they already have the options for holobarriers from atmos to let them do it without causing breaches.

**Comment by GunHog** [comment]
What if the shuttle came with a construction console, like the Aux Base has? You would not even need to pre-make the walls, then, just extra mats to feed the console!

**Comment by WJohn** [comment]
That'd be a pretty cool idea, though I'd have to coordinate it with you if you want to try that instead.

**Comment by Farquaar** [comment]
Could this be added in as an alternative to the original? I've had some awesome memories with the mega BYOS kit. We ended up building a fully functional medbay, a garden, a bar, a command/security wing, and an engineering supply centre. It looked fantastic and was a blast to make.

**Comment by zxaber** [comment]
Truth be told, half the fun of the DIY shuttle for me is setting up atmos. The rest is just adding some chairs and _maybe_ internal walls. Honestly, we have EVA suits for a reason.

The extra equipment in the center is nice, though.

**Comment by imsxz** [comment]
i feel like this kinda defeats the purpose of the BYOS. it's supposed to be a challenge shuttle and ive seen lots of really nice things done with the massive space on longer rounds. In this state it's just free 7500 cargo credits for an otherwise fine and normal shuttle, the only downside being that antags can destroy it before it docks.

I don't agree that the massive size of the original was a bad thing either, I've gotten around it personally by just building a smaller walled-in portion in an emergency situation and left the rest of the shuttle untouched and it worked out fine.

tl;dr the shuttle is supposed to be a challenge not appealing that's why it gives you money

**Comment by Mickyan** [comment]
You could give it minimal atmosphere sorta like lavaland so setting up proper atmos is still favorable without killing anyone without a spacesuit

**Comment by AffectedArc07** [comment]
Can we make the shuttle take 10 minutes to get to the station when you order it. Then it comes with a shuttle console onboard, and then you can use that to start a 3 minute launch timer

**Comment by FloranOtten** [comment]
> Can we make the shuttle take 10 minutes to get to the station when you order it. Then it comes with a shuttle console onboard, and then you can use that to start a 3 minute launch timer

So just a normal escape shuttle, then?

**Comment by TerraGS** [comment]
I think this can exist along side the current BYOS for a higher cost but still lower than standard. You could call it the "Some Assembly Required Shuttle."

**Comment by anconfuzedrock** [comment]
>RD main buys the BYOS so he can get free gloves

**Comment by fludd12** [comment]
Personally, I highly suggest the construction console idea.  With a few modifications, you could even allow the construction console to place and remove engine tiles, which could make things look a lot nicer. This still gives the possibility of the shuttle needing to be launched and not having any atmospherics, but I feel like that's what you get for a free 7500 cargo points.

If you're feeling particularly adventurous, you could have a system that saves BYOS designs that carry >50% of a server population for a while, and give players an option to purchase a discounted "Corporate-Union Designed Shuttle" that selects from some of the saved ones randomly.

That's going a bit ahead of the scope of this PR, of course; just firing ideas into the void.

**Comment by WJohn** [comment]
I feel that as the shuttle is now with this proposal, I should probably knock the refund down from -7500 to maybe -5000 or even -2500. since it's a much more usable shuttle even if no construction happens.

What do you guys think?

**Comment by wesoda25** [comment]
Yeah -2500 sounds good

**Comment by zxaber** [comment]
Honestly, I think I would like this shuttle to be it's own option and not replace the DIY. This shuttle needs nothing done to it to carry people out. I'm not saying that people won't work on it, but the DIY's bareness that requires setting up walls, and running pipes and vents and air alarms for atmos, is what makes it fun.

**Comment by anconfuzedrock** [comment]
also honestly I will see captains buying this shuttle every round if it has pressure since there'd be no downside to doing so its free money for a large shuttle

**Comment by WJohn** [comment]
The downside is that it has no brig and no console to launch early and it can be fucked up by anyone way ahead of time and there's no protection/segregation for any of the heads of staff who board it unless it gets built.

**Comment by anconfuzedrock** [comment]
No shuttle console? So you’re saying it’s hijack proof?
I legitimately thing the buffed BYOS will be op if it doesn’t cost money.

**Comment by anconfuzedrock** [comment]
I was gonna go on a rant about shuttles as a whole but here’s the important idea:
Most shuttles with a specific purpose cost money, the purpose of scrapheap is to give money. This shuttle has a very specific purpose. Make this one cost at least 2500 or something, but give it more supplies. (or just leave its current iteration the cap can just open eva and cargo lobby with autolathe anyways)
That way caps would only buy it if they wanted the buildy part and not the “this shuttle is super good” part (well maybe they still would its pretty big 2500 is a great deal) its not like 2500 money will get in the way of a byos.
Tldr**MAKE IT COST MONEY BUT WITH MORE STUFF**

**Comment by optimumtact** [comment]
Im not sure I really understand this change?

**Comment by improvedname** [comment]
>free stack of titanium, >free pair of insulated gloves, >electrical toolbox with a chance of insulated spawn in, gUess im bUying this EveRy round

**Comment by optimumtact** [comment]
you're overselling that

**Comment by Timonkeyn** [comment]
The new assistant meta is to storm this for the sweet insuls when it gets bought

**Comment by optimumtact** [comment]
There are random spawns of insuls on maps anyway, so that's hardly a major issue. I guess I just dont' understand why it needs a buff, is it being ordered as a grief shuttle often?

**Comment by zxaber** [comment]
Feels like a nerf to me, actually. The fun parts of the DIY shuttle get removed and it becomes an "I dunno, maybe go add chairs to the shuttle if you're bored" purchase.

**Comment by optimumtact** [comment]
I guess we can try this out and if we want we can add a harder DIY option again
